istioctl/pkg/wait/wait.go

```diff
@@ -61,6 +62,9 @@ func Cmd(cliCtx cli.Context) *cobra.Command {
 		Example: `  # Wait until the bookinfo virtual service has been distributed to all proxies in the mesh
   istioctl experimental wait --for=distribution virtualservice bookinfo.default
 
+  # Wait until the bookinfo virtual service has been distributed to a specific proxy
+  istioctl experimental wait --for=distribution virtualservice bookinfo.default --proxy workload-instance.namespace
```

Comment:
The change makes sense to me, but I think we have not used a similar flag before. For other commands, we directly use proxyID as args instead of `--proxy`.

Reply:
/cc @therealmitchconnors to take a look.

Reply:
Good suggestion